schema/workflow.yaml

```diff
@@ -165,34 +203,42 @@ $defs:
             type: string
             const: asyncapi
           with:
-            title: WithAsyncAPI
             type: object
+            title: WithAsyncAPI
```

Comment:
The title should be `AsyncApiArguments` or something like that IMHO: `WithAsyncApi` doesn't describe the object and might even be confusing.